src/scripts/clipperUI/mainController.tsx

```diff
@@ -256,6 +257,10 @@ export class MainControllerClass extends ComponentBase<MainControllerState, Main
 			case PanelType.RegionInstructions:
 				return <RegionSelectingPanel clipperState={this.props.clipperState} />;
 			case PanelType.ClippingToApi:
+				if (this.props.clipperState.currentMode.get() === ClipMode.Pdf) {
+					return <ClippingPanelWithDelayedMessage clipperState={this.props.clipperState}
```

Comment:
We should do this as a separate change, but I'd like to add a more generic version of this for any of the non-PDF clips that take more than a few seconds to come back.  Filed #116 to track.